Algolia.Search/Index.cs

```diff
@@ -172,6 +172,37 @@ public Task<JObject> GetObjectsAsync(IEnumerable<String> objectIDs, Cancellation
         }
 
         /// <summary>
+        /// Get several objects from this index.
+        /// </summary>
+        /// <param name="objectIDs">An array of unique identifiers of the objects to retrieve.</param>
+        /// <returns></returns> 
+        public Task<JObject> GetObjectsWithAttributesAsync(IEnumerable<String> objectIDs, IEnumerable<string> attributesToRetrieve, CancellationToken token = default(CancellationToken))

```

Comment:
You can have two prototypes for the same method name in csharp no?

`public Task<JObject> GetObjects(IEnumerable<String> objectIDs, IEnumerable<string> attributesToRetrieve, CancellationToken token = default(CancellationToken))`
and
`public Task<JObject> GetObjects(IEnumerable<String> objectIDs, IEnumerable<string> attributesToRetrieve, CancellationToken token = default(CancellationToken))`